src/bin/tools/jar.ts

```diff
@@ -90,5 +90,5 @@ if (require.main === module) {
             groupId: process.env.GROUP_ID ?? "group",
             version: process.env.VERSION ?? "1.0.0"
         });
-    main().catch(e => console.error(e));
```

Comment:
Catching this error was causing keycloakify to swallow errors & instead report a successful message & error code:
![image](https://user-images.githubusercontent.com/10997562/228699946-0f22e664-2b18-4f3a-a9d2-ab8ab17416f8.png)

Reply:
Yes 💯  you would **never** catch me doing this, I am a vocal advocate for fail early strategy, I just didn't review.

Reply:
oh damn, that will affect other placeses too